pkgs/top-level/python-packages.nix

```diff
@@ -2327,8 +2331,6 @@ in {
 
   pyhcl = callPackage ../development/python-modules/pyhcl { };
 
-  pyhepmc = callPackage ../development/python-modules/pyhepmc { };
```

Comment:
at the very least add a deprecation mechanic:
```
pyhepmc = throw "please use hepmc3";
```

Reply:
Thanks for reminding me about this. I've now considered this and I don't think this is needed in this particular case.

Reply:
the other option would be to alias, but I don't like aliasing in python packages, they should reflect the name used on pypi (if available)